Assets/GothicVR/Scripts/Data/SkyState.cs

```diff
@@ -0,0 +1,184 @@
+using UnityEngine;
+
+namespace GVR.Data
+{
+    public class SkyState
+    {
+        public float time;
+        public Vector3 polyColor;
+        public Vector3 fogColor;
+        public Vector3 domeColor1;
+        public Vector3 domeColor0;
+        public float fogDist = 0;
+        public int sunOn = 1;
+        public int cloudShadowOn = 0;
+        public SkyLayerData[] layer;
+
+
+        public SkyState()
+        {
+            layer = new SkyLayerData[2];
+            layer[0] = new SkyLayerData();
+            layer[1] = new SkyLayerData();
+        }
+
+
+        public void PresetDawn()
```

Comment:
Please add some comment (e.g. on class level), that all of these values are based on Gothic 1's official data. Otherwise people might ask, why polyColor is 190, 160, 225 etc.